src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Helpers.cs

```diff
@@ -11,24 +11,24 @@ namespace System.Text.Json
 {
     public sealed partial class Utf8JsonWriter
     {
-        private const byte LengthMask =         0b000_000_11;
-        private const byte EncodingMask =       0b000_111_00;
+        private const byte PartialCodePointLengthMask =         0b000_000_11;
```

Comment:
I would try to collocate these constants with the definition of `_partialCodePointFlags` so that future readers can better understand how it works.